source/d1min.h

```diff
@@ -24,12 +24,14 @@ class D1Min : public QObject {
     quint16 getSubtileCount();
     quint16 getSubtileWidth();
     quint16 getSubtileHeight();
-    QList<quint16> &getCelFrameIndices(quint16 subTileIndex);
+    QList<quint16> &getCelFrameIndices(int subTileIndex);
+    void createSubtile();
+    void removeSubtile(int subTileIndex);
 
 private:
     QString minFilePath;
     D1Gfx *cel = nullptr;
-    quint8 subtileWidth;
-    quint8 subtileHeight;
+    quint8 subtileWidth = 2;
+    quint8 subtileHeight = 5;
```

Comment:
is this revert intentional?

Reply:
Well, kind of. I'm not exactly sure yet how to implement the 'New CEL/CL2/Tileset' menu options. Somewhere a reasonable default values have to be set to reduce the necessary input from the user. So I thought it could be set there for the moment.